io/pci/PowerVMEEH.py

```diff
@@ -60,11 +60,13 @@ def setUp(self):
         if output != '0x1':
             self.skip("EEH is not enabled, please enable via FSP")
             sys.exit(1)
-        self.max_freeze = int(self.params.get('max_Freeze', default='1'))
+        self.max_freeze = int(self.params.get('max_freeze', default='1'))
```

Comment:
i think it should be` maxFreeze` as yaml entry show this , please rectify in any one of the place